src/templates/header.html

```diff
@@ -27,21 +27,35 @@
     <a class="navbar-brand" href="{{ url_for('.root') }}">
         {{ title }}
     </a>
+    <div class="navbar-nav mr-auto">
+        {% if is_staff %}
+            {% if "staff" in request.path %}
+                <a href="{{ switcher_endpoint(course, assignment) }}" style="margin-right: 10px;">
+                    <button class="btn btn-primary">Staff</button>
```

Comment:
We can add a "title" attribute to describe what this button is used for so user can see it when hovering their mouse over it for some time. 

```
<button class="btn btn-primary" title="Switch to student view">Staff</button>
```
(The mouse is not in this screenshot but it was there)
![image](https://user-images.githubusercontent.com/31719253/93279446-e5bbe580-f78c-11ea-9cbf-641d0452751c.png)

Not the most obvious instruction set but hopefully confuse the user a little less.

Reply:
Added, thanks! I was looking into changing the text on hover but that seems like it'll require some CSS magic, plus it wouldn't be mobile-friendly. We can easily change the text to be clearer later if necessary